WordPress/src/main/res/values/strings.xml

```diff
@@ -970,6 +970,9 @@
     <string name="aztec_editor_enabled" translatable="false">Aztec Editor enabled</string>
     <string name="aztec_editor_disabled" translatable="false">Aztec Editor disabled</string>
     <string name="aztec_editor_available" translatable="false">Aztec Editor available</string>
+    <string name="new_editor_promo_button">@android:string/ok</string>
+    <string name="new_editor_promo_description">The WordPress app now includes a beautiful new editor. Try it out by creating a new post.</string>
+    <string name="new_editor_promo_title">New editor</string>
```

Comment:
Did we want to use a different title for this editor? I don't know if the `Aztec` name is a brand or not, but something to consider.